runtime/pom.xml

```diff
@@ -167,6 +167,11 @@
             <version>6.0.53</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
+        </dependency>
```

Comment:
Wouldn't this cause an issue if the class path already has slf4j bindings ?

Reply:
Removed for now